docfx.json

```diff
@@ -907,10 +907,21 @@
             },
             "ms.update-cycle": {
                 "docs/ai/**/**.{md,yml}": "180-days",
-                "docs/csharp/whats-new/**/**.{md,yml}": "1825-days",
-                "docs/framework/**/**.{md,yml}": "3650-days",
+                "docs/csharp/whats-new/**/**.{md,yml}": "1095-days",
+                "docs/csharp/advanced-topics/interop/**/**.{md,yml}": "1825-days",
+                "docs/csharp/advanced-topics/reflection-and-attributes/**/**.{md,yml}": "1825-days",
+                "docs/csharp/fundamentals/exceptions/**/**.{md,yml}": "1825-days",
+                "docs/csharp/language-reference/**/**.{md,yml}": "1095-days",
+                "docs/csharp/language-reference/compiler-messages/**/**.{md,yml}": "3650-days",
```

Comment:
reference for compiler errors and warnings. Any changes would be considered a breaking change, so it's very unlikely that these change. (New errors are added in every release)